### DISCUSSION TITLE
set an expiration date

### DISCUSSION THREAD

**Discussion by gmelendez** [josdejong/svelte-jsoneditor#362] [Ideas]
Hi JsonEditorOnline team,

Can you please consider to add a way to set an expiration date for the request the user is sharing?

![image](https://user-images.githubusercontent.com/403855/180567347-11024048-5324-44d5-8e23-4e4f0b4e23a7.png)

Thanks in advance

**Comment by josdejong** [comment]
Thanks for your suggestion. That is an interesting idea.

And maybe the documents should be cleaned up by default after they have not been accessed for say more than a year or so.

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it.
